packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.1.11
+
+* Added flag to generate null safe Dart code `--dart-null-safety`.
```

Comment:
I would call this `null safety aware` code rather than `null safe` no?

Reply:
The feature of the annotations is called "null safety".  Even if you are using a ?-type you are getting extra safety by enforcing explicit casting to non-?-types.  If we support non-null types in the generated code it would still be generated via this flag.

Reply:
The flag sounds good. I'd leave that as is. I just meant the `generate null safe` part. I'd argue a non-nullable type being passed through the function is "null safe". Unsafe casting a nullable type is "null safety aware". 

i.e. I wouldn't make it sound like the user can just run pigeon once in v0.1.11, have "null safe" code and never have to think about it or regenerate from a future version again since it sounds like it's "done".

Reply:
Oh, i see what you are saying.  I switched it to be called "null safety annotated".